weather/tests/test_data_funcs.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+from data_funcs import isfloat
+from data_funcs import to_float
```

Comment:
Once cairo is a package you can use the format `from package.module import function` to import these. Right now these don't load.